Webmin/input/checks/templates/file_dir_permissions.csv

```diff
@@ -0,0 +1,3 @@
+/etc,shadow,0,0,0000
+/etc,gshadow,0,0,0000
+/etc,passwd,0,0,0644
```

Comment:
does the webmin content need to check file perms on these files, or is this an artifact of the directory structure copy/import?